### ISSUE TITLE
Gamecube-Controller-Input with an adapter?

### ISSUE THREAD

**Issue by Menki1** [Maschell/hid_to_vpad#100]
Just a little problem I have: I'm using an Gamecube-Controller-Adapter for the wiiu (A noname one) and hid t vpad did not find it? Is there a way to fix this little issue of mine?

**Comment by Andytheclown** [comment]
yes, you can make your own configuraion file https://github.com/Maschell/controller_patcher/wiki/2.-Gamepad-button-remapping
